docs/index.rst

```diff
@@ -81,7 +81,20 @@ Reading data: Statuses
 These functions allow you to get information about single statuses.
 
 .. automethod:: Mastodon.status
+
+Returns a single toot dict for the given status.
+
 .. automethod:: Mastodon.status_context
+
+.. code-block:: python
+    mastodon.status_context(<numerical id>)
+    # Returns
+    {
+     'descendants': A list of toot dicts
+     'ancestors': A list of toot dicts
+    }
+Note that this can only be called on statuses belonging to the currently logged-in user.
```

Comment:
I think the problem is you're trying with an ID like what's in the URLs on the web client, which is a _different_ id from what is in the API. That, as well, should probably be documented somewhere, to avoid people tripping over it (I sure did).